h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -41,26 +41,31 @@ class SearchBarController extends Controller {
       }.bind(this), 0);
     };
 
-    var getPreviousSiblingElement = element => {
+    var getPreviousVisibleSiblingElement = element => {
       if (!element) {
         return null;
       }
 
       do {
         element = element.previousSibling;
-      } while (element && element.nodeType != 1);
-
+      } while (element &&
+        (element.nodeType !== 1 ||
+          !element.classList ||
+          element.classList.contains('is-hidden')));
```

Comment:
It's not super obvious what this boolean expression means and it's repeated twice, might be worth moving it into a helper function named something like `isVisible(element)`?